packages/react/src/VideoPlayer/VideoPlayer.features.stories.tsx

```diff
@@ -11,14 +12,85 @@ export default {
 
 export const WithPoster = () => (
   <VideoPlayer poster={posterImage} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
   </VideoPlayer>
 )
 
 export const WithoutBranding = () => (
-  <VideoPlayer branding={false} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+  <VideoPlayer showBranding={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithoutTitle = () => (
+  <VideoPlayer showTitle={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithCustomControls = () => (
+  <VideoPlayer
+    title="GitHub media player"
+    renderControls={() => (
+      <ControlsBar>
+        <PlayPauseButton />
+        <SeekControl />
+      </ControlsBar>
+    )}
+  >
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const HideControlsUntilPlaying = () => (
+  <VideoPlayer
+    title="GitHub media player"
+    renderControls={(controlsProps, {isPlaying}) => (
+      <Controls
+        style={{
+          opacity: isPlaying ? 1 : 0,
+          transition: 'opacity var(--brand-animation-duration-default) var(--brand-animation-easing-default)',
+        }}
+        {...controlsProps}
+      />
+    )}
+  >
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithCustomPlayButton = () => (
+  <VideoPlayer
+    title="GitHub media player"
+    renderPlayOverlay={() => <PlayIcon style={{width: 96, height: 96, opacity: 0.8}} />}
```

Comment:
This should be the default play button tbh 😅... looks much nicer

Reply:
I agree 😄

Reckon I should change it?

Reply:
I think so... 

@simmonsjenna @danielguillan Cool if we change the main play icon?

 
<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2024-07-16 at 16 06 03](https://github.com/user-attachments/assets/d1f343a4-1c54-4142-bd57-55335779693c)


 </td>
<td valign="top">

![Screenshot 2024-07-16 at 16 06 08](https://github.com/user-attachments/assets/4ba5281d-6008-46b1-b91a-b207a31c22c8)


</td>
</tr>
</table>

Reply:
@rezrah @joshfarrant Agreed — let's change it.

Reply:
@rezrah @joshfarrant As a follow up, the newer one is a bit large — can we resize it across certain key breakpoints so it's more proportional as the video player gets smaller?